exchanges/binance/binance.go

```diff
@@ -652,6 +654,17 @@ func (b *Binance) GetAccount() (*Account, error) {
 	return &resp.Account, nil
 }
 
+func (b *Binance) GetMarginAccount() (*MarginAccount, error) {
```

Comment:
I guess this is tested `TestGetAccountInfo` in `binance_test.go`? I added a `asset.Margin` case in those commits. Of course I keep just currency holdings and discard everything else since the account types is pretty generic. The test passed on my computer.

Reply:
Indeed! You are right!